vcpkg/concepts/glossary.md

```diff
@@ -58,3 +103,30 @@ A vcpkg-specific term, a port contains:
 * Instructions to acquire, build if necessary, and install the package.
 
 See the [ports documentation](../concepts/ports.md) to learn more.
+
+## S
+
+### System Package Manager
+
+A package manager such as apt-get designed to be used with a particular
+platform such as Linux, or brew for MacOS. System package managers install
+packages and libraries system wide, instead of for a particular project.
+
+## T
+
+### Triplet
+
+A set of properties or script applied to all libraries before vcpkg builds,
+essentially a profile. A triplet specifies the linkage type, build flags,
```

Comment:
The problem with this definition is that it introduces a new term, what is a "profile"?